packages/sync-engine/src/supabase/edge-functions/stripe-setup.ts

```diff
@@ -1,11 +1,14 @@
 import { StripeSync, runMigrations, VERSION } from 'npm:stripe-experiment-sync'
 import postgres from 'npm:postgres'
 
+// Get management API base URL from environment variable (for testing against localhost/staging)
+const MGMT_API_BASE = Deno.env.get('SUPABASE_MANAGEMENT_URL') || 'api.supabase.com'
+
 // Helper to validate accessToken against Management API
 async function validateAccessToken(projectRef: string, accessToken: string): Promise<boolean> {
   // Try to fetch project details using the access token
   // This validates that the token is valid for the management API
-  const url = `https://api.supabase.com/v1/projects/${projectRef}`
+  const url = `https://${MGMT_API_BASE}/v1/projects/${projectRef}`
```

Comment:
Even though it will likely always be using `https`, might as well include the protocol in the param